### ISSUE TITLE
On after signup edit page: form validation not working properly

### ISSUE THREAD

**Issue by 1000miles** [sozialhelden/wheelmap#175]
**Case:**
* I am logged in 
* I am on after signup edit page (`/profile/after_signup_edit`)
* I leave the email input field empty (even though it is marked as required) or I type in a non-valid email address with `dfdgfdgdg`
* I click on `Fertigstellen` button

=> I should get an error message which tells me to provide an email address first in order to complete the signup process, instead I get the success message: `Du bekommst in wenigen Minuten eine E-Mail mit einem Link, um deine E-Mail-Adresse zu bestätigen.`

**In the browser inspector I can find this:**
`<div id="user_email_input" class="email input required stringish form-group">`

@lennerd It seems the form validation doesn't work properly? 

<img width="1275" alt="after_signup_edit_page_message" src="https://cloud.githubusercontent.com/assets/4077916/14241838/24fb682e-fa4e-11e5-88fe-db3160c5db4f.png">

**Comment by lennerd** [comment]
Yeah, that seems to be a bug. I will take a look at this as soon as #105 is deployed.

**Comment by 1000miles** [comment]
:+1: Thank you for looking into it.

**Comment by lennerd** [comment]
Do you created a new account or did you login with an existing one?

**Comment by holgerd** [comment]
on hold, waiting for PR #359 to review.
